src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Microsoft.VisualStudio.LanguageServices.Razor.Test.csproj

```diff
@@ -39,6 +39,6 @@
 
   <ItemGroup>
     <!-- Package changes to enable dev17 builds that have incoherent versions -->
-    <PackageReference Include="Microsoft.IO.Redist" Version="4.7.1" />
+    <PackageReference Include="Microsoft.IO.Redist" Version="6.0.0" />
```

Comment:
IIRC you can just remove this explicit reference now because the versions are no longer incoherent in any versions we test against. If that's wrong and you can't remove it please use a variable for the version number.